### ISSUE TITLE
No branding assets available for use

### ISSUE THREAD

**Issue by konobi** [joyent/sdc#169]
There doesn't appear to be any brand assets (logos, images, etc) for SDC/Triton that are available for the community to use.

Is there a plan for this?

**Comment by misterbisson** [comment]
I don't think we had a plan until you asked, but it makes sense. I'll see what I can do.

**Comment by tsantero** [comment]
subscribe

**Comment by RAWRAW** [comment]
All - the marketing team is working to quickly pull this together for general use in the community. Thanks! RAW

**Comment by lloydde** [comment]
@RAWRAW update?

**Comment by tgross** [comment]
@misterbisson @ShubhraKar this is a bit of an old issue but do we have updates on this?
